### PR TITLE
Redirect duplicate readme files

### PR DESCRIPTION
**Pull request by kaustavghosh06**
Redirecting duplicate files:

Redirect react tutorial -> https://github.com/aws-amplify/docs/blob/master/js/react.md
Redirect GraphQL tutorials -> https://github.com/aws-amplify/docs/blob/e02bd04e09cac735ddc2ee66a3141e2ce9663302/js/graphql.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: amplifycli_react_tutorial.md [hunk 1]
```diff
@@ -1,288 +1 @@
...
-
-```
-amplify function invoke <resourcename>
-```
-
-In this case, the function runs, but it doesn't exit because this Lambda example starts an Express server which you need to manually close when testing it from the CLI. Use `ctrl-c` to close and open the `./amplify/backend/function/resourcename` directory to see the local structure that is packaged for Lambda invocation from API Gateway. The Lambda function is inside the `src` directory, along with `event.json`, which is used for the `amplify function invoke` command you just ran. `index.js` is also in this directory, which is the main entry point for the Serverless Express library that echoed out the test event and instantiated the server inside `app.js`. Since the Express routes defined in `app.js` don't have a path that's called via the test event, it responded with a 404 message. For more information, see https://github.com/awslabs/aws-serverless-express.
+<meta http-equiv="refresh" content="0; URL='https://github.com/aws-amplify/docs/blob/master/js/react.md'" />
```
Comment (mlabieniec) [inline amplifycli_react_tutorial.md]: This should change to redirecting to the actual GH site:

https://aws-amplify.github.io/aws-amplify/docs/js/react

File: how-to-write-a-transformer.md [hunk 2]
```diff
@@ -1,516 +1 @@
...
-                ...type,
-                fields: updatedFields
-            }
-            ctx.putType(updatedType)
-        }
-    }
-}
-```
\ No newline at end of file
+<meta http-equiv="refresh" content="0; URL='https://github.com/aws-amplify/docs/blob/master/js/graphql.md'" />
```
Comment (mlabieniec) [inline how-to-write-a-transformer.md]: same as above, make sure these redirect to the actual compiled site:

https://aws-amplify.github.io/aws-amplify/docs/js/graphql

**Comment by kaustavghosh06** [comment]
@mlabieniec Fixed the paths.

**Comment by kaustavghosh06** [comment]
@mlabieniec Made the requested changes.

**Pull request merged**